CONTRIBUTING.md

```diff
@@ -177,7 +177,7 @@ subsequent comment to the PR.
 ### Translation changes
 
 Note that translations should not be submitted as pull requests. Please see
-[Translation Process](https://github.com/bitcoin/bitcoin/blob/master/doc/translation_process.md)
+[Translation Process](https://github.com/bitcoinknots/bitcoin/blob/master/doc/translation_process.md)
```

Comment:
We need to actually have a translation process first.

Core stole the Transifex repo, so the older process no longer works for us